compiler/lineinfos.nim

```diff
@@ -96,6 +96,7 @@ const
     warnProveInit: "Cannot prove that '$1' is initialized. This will become a compile time error in the future.",
     warnProveField: "cannot prove that field '$1' is accessible",
     warnProveIndex: "cannot prove index '$1' is valid",
+    warnTrailingElse: "trailing else, all cases are already covered",
```

Comment:
I think I'd speak more to the problem you're warning about; "else clause ignored" or some such.

Reply:
Renamed it to UnreachableElse now.